lib/user.js

```diff
@@ -181,7 +181,7 @@ function toExternalUser(user) {
  */
 function toInternalUser(user) {
     for (var prop in user) {
-        if ( prop === 'roleTenantFilter' && user[prop] !== null ) {
+        if ( prop === 'roleTenantFilter' && user[prop] !== null && typeof(user[prop]) !== 'string' ) {
```

Comment:
Added for backward compatibility, as the current workaround is to use API-compatible **string** when calling AM user creation method, e.g. `roleTenantFilter: 'ECOM_USER:abcd_dev,abcd_stg'`